src/content/nft/index.md

```diff
@@ -316,56 +315,16 @@ When you mint an NFT, a few things have to happen:
 - The transactions that confirm the above need to be added to a block and "immortalised" on the chain.
 - The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximise their earnings.
```

Comment:
```suggestion
- The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximize their earnings.
```